src/libs/Notification/LocalNotification/types.ts

```diff
@@ -1,26 +1,16 @@
-import {ImageSourcePropType} from 'react-native';
-import {OnyxEntry} from 'react-native-onyx';
 import {Report, ReportAction} from '@src/types/onyx';
 
-type PushParams = {
-    title: string;
-    body?: string;
-    icon?: string | ImageSourcePropType;
-    delay?: number;
-    onClick?: () => void;
-    tag?: string;
-};
+type LocalNotificationClickHandler = () => void;
 
-type ReportCommentParams = {
-    report: OnyxEntry<Report>;
-    reportAction: ReportAction;
-    onClick: () => void;
+type LocalNotificationData = {
+    reportID?: string;
 };
 
 type LocalNotificationModule = {
-    showCommentNotification: (reportCommentParams: ReportCommentParams) => void;
+    showCommentNotification: (report: Report, reportAction: ReportAction, onClick: LocalNotificationClickHandler) => void;
     showUpdateAvailableNotification: () => void;
-    showModifiedExpenseNotification: (reportCommentParams: ReportCommentParams) => void;
+    showModifiedExpenseNotification: (report: Report, reportAction: ReportAction, onClick: LocalNotificationClickHandler) => void;
+    clearReportNotifications: (reportID: string) => void;
```

Comment:
```suggestion
    clearReportNotifications: ClearReportNotifications;
```

NAB. Use the exported type from ClearReportNotifications